README.md

```diff
@@ -45,22 +44,21 @@ jupyter labextension list
 
 It is recommended to first install JLNE and all of its dependencies on a fresh environment to ensure it is stable.
 
-1) Start by installing ![jupyterlab](https://jupyter.org/install) and all of its dependencies.
-2) Install ipykernel
+1) Install ![jupyterlab](https://jupyter.org/install), ipykernel (conda_env is your current environment name), ![NodeJS](https://nodejs.org/en/download/package-manager/), and pyyaml.
 ```
+conda install jupyterlab
```

Comment:
I'd actually prefer you consolidate all the conda installation into this -- move pyyaml and nodejs up here.